test/routes.spec.js

```diff
@@ -135,4 +135,17 @@ describe('API Routes', () => {
       });
     });
   });
+  describe('/api/v1/playlists', () => {
+    it("getting response from api/v1/playlists", done => {
```

Comment:
You are also testing that playlists are returned, in addition to getting a 200 response, consider an it block description such as,  `it("returns all playlists"....) Of course, whatever is consistent with your team standard for describing test blocks is the priority.